concepticondata/conceptlists/Lastra-1986-381.tsv

```diff
@@ -0,0 +1,382 @@
+Lastra-1986-381-195	195	sepulcro	tomb	1773	GRAVE
+Lastra-1986-381-196	196	rojo	red	156	RED
+Lastra-1986-381-197	197	negro	black	163	BLACK
+Lastra-1986-381-198	198	blanco	white	1335	WHITE
+Lastra-1986-381-199	199	chico	boy	1366	BOY
+Lastra-1986-381-200	200	chaparro	short	2484	SMALL (NOT TALL)
+Lastra-1986-381-201	201	alto (una persona)	High (a person)	3346	TALL (PERSON)
+Lastra-1986-381-202	202	llano (un terreno)	plain (land)	652	PLAIN
+Lastra-1986-381-203	203	puntiagudo (un palo)	pointed (a stick)	372	POINTED
+Lastra-1986-381-204	204	filoso	Filoso	1396	SHARP
+Lastra-1986-381-205	205	redondo (cualquier palabra que se obtenga preguntar si hay otra)	round (any word that is obtained by asking if there is another)	1395	ROUND
+Lastra-1986-381-206	206	vacío (una botella)	empty	1624	EMPTY
+Lastra-1986-381-207	207	frío (agua)	Cold (water)		
+Lastra-1986-381-208	208	caliente (agua)	hot (water)		
+Lastra-1986-381-209	209	espeso (el atole)	thick (the atole)		
+Lastra-1986-381-210	210	aguado (el atole)	watery (the atole)		
+Lastra-1986-381-211	211	bueno (persona)	good (person)		
+Lastra-1986-381-212	212	bueno (sabor)	good (flavour)		
+Lastra-1986-381-213	213	borracho (winti, tlawanki: cualquiera que den, preguntar la otra)	Drunk (Winti, Tlawanki: Anyone to ask, ask the other)	1463	DRUNK
+Lastra-1986-381-214	214	jorobado	hunchback	99	HUNCHBACK
+Lastra-1986-381-215	215	sordo	deaf	996	DEAF
+Lastra-1986-381-216	216	cansado	tired	1757	TIRED
+Lastra-1986-381-217	217	mano izquierda	left hand	2182	LEFT HAND
+Lastra-1986-381-218	218	sucia (la ropa)	Dirty (clothes)		
+Lastra-1986-381-219	219	nuevo	new	1231	NEW
+Lastra-1986-381-220	220	rápido	fast	1631	FAST
+Lastra-1986-381-221	221	crudo (carne)	raw (meat)	1959	RAW
+Lastra-1986-381-222	222	triste	sad	699	SAD
+Lastra-1986-381-223	223	cosquilloso	ticklish		
+Lastra-1986-381-224	224	débil (una persona)	weak (a person)		
+Lastra-1986-381-225	225	débil (una pared, un lazo)	weak (a wall, a bond)		
+Lastra-1986-381-226	226	delgado (una tabla)	thin (a table)	2249	THIN (OF SHAPE OF OBJECT)
+Lastra-1986-381-227	227	grueso (una tabla)	thick (a table)	1244	THICK
+Lastra-1986-381-228	228	duro	hard	1884	HARD
+Lastra-1986-381-229	229	derecho (un camino)	straight (a path)	1404	STRAIGHT
+Lastra-1986-381-230	230	acostado	lying down	1411	LIE (REST)
+Lastra-1986-381-231	231	sentado	sitting	1416	SIT
+Lastra-1986-381-232	232	parado	stand	1442	STAND
+Lastra-1986-381-233	233	boca abajo	face down		
+Lastra-1986-381-234	234	éste	this	1214	THIS
+Lastra-1986-381-235	235	ése	that	78	THAT
+Lastra-1986-381-236	236	aquél	that	78	THAT
+Lastra-1986-381-237	237	así	So		
+Lastra-1986-381-238	238	aquí	here	136	HERE
+Lastra-1986-381-239	239	cerca	near	1942	NEAR
+Lastra-1986-381-240	240	hoyo	hole		
+Lastra-1986-381-241	241	pasado mañana	day after tomorrow	1179	DAY AFTER TOMORROW
+Lastra-1986-381-242	242	antier	the day before yesterday	1180	DAY BEFORE YESTERDAY
+Lastra-1986-381-243	243	temprano ('en la mañana temprano')	Early ('early morning')		
```

Comment:
Would this rather be `672 EARLY` as opposed to the one below?

Reply:
But not sure here. :)

Reply:
I believe that the gloss below should receive the mapping (based on the definition offered in Concepticon) :)